libs/shared/src/utils/bridge.utils.ts

```diff
@@ -19,14 +19,22 @@ type InAppOutput = {
   avatar?: string;
   primaryAction?: {
     label: string;
-    url?: string;
+    redirect?: Redirect;
   };
   secondaryAction?: {
     label: string;
-    url?: string;
+    redirect?: Redirect;
   };
+  redirect?: Redirect;
 };
 
+type Redirect = {
+  url: string;
+  target?: Target;
+};
+
+type Target = '_self' | '_blank' | '_parent' | '_top' | '_unfencedTop';
+
```

Comment:
We can use a shared typing declared in `messages.interface.ts` for these, to avoid repetition and improve maintainability (see above comment).